client-sdk/go/modules/evm/evm.go

```diff
@@ -0,0 +1,76 @@
+package evm
+
+import (
+	"context"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/client"
+)
+
+const (
+	// Callable methods.
+	methodCreate = "evm.Create"
+	methodCall   = "evm.Call"
+
+	// Queries.
+	methodPeekStorage = "evm.PeekStorage"
+	methodPeekCode    = "evm.PeekCode"
+)
+
+// V1 is the v1 EVM module interface.
+type V1 interface {
+	Create(value []byte, initCode []byte, gasPrice []byte, gasLimit uint64) *client.TransactionBuilder
+	Call(address []byte, value []byte, data []byte, gasPrice []byte, gasLimit uint64) *client.TransactionBuilder
+	PeekStorage(ctx context.Context, address []byte, index []byte) ([]byte, error)
+	PeekCode(ctx context.Context, address []byte) ([]byte, error)
```

Comment:
Add documentation comments similar to other modules.

Reply:
Done :)